barman/cli.py

```diff
@@ -484,6 +484,15 @@ def backup_completer(prefix, parsed_args, **kwargs):
             default=None,
             type=check_non_negative,
         ),
+        argument(
+            "--keepalive-interval",
+            help="An interval, in seconds, at which a heartbeat query will be sent "
+            "to the server to keep the connection alive during an Rsync backup. "
```

Comment:
"keep the libpq connection alive", so nobody thinks this is related to the rsync connections

Reply:
good point